kernelci/cli.py

```diff
@@ -165,6 +165,11 @@ class Args:
         "Path to the install directory, or _install_ inside kdir by default",
     }
 
+    is_admin = {
```

Comment:
Please mention in the commit subject that the arguments are for creating API users.

Reply:
Done.